### DISCUSSION TITLE
pnpm lowdefy dev Starting lowdefy 4.0 keeps telling me that I'm missing prerender-manifest.json. Why?

### DISCUSSION THREAD

**Discussion by xwen107** [lowdefy/lowdefy#1598] [Ideas and feature requests]
pnpm lowdefy dev Starting lowdefy 4.0 keeps telling me that I'm missing prerender-manifest.json. Why?
`14:10:24 ✖ node exited with code 1
14:10:24 ∙ ready - started server on 0.0.0.0:3000, url: http://localhost:3000
14:10:25 ✖ Error: Cannot find module 'C:\Users\Xqw\Desktop\example\app\.lowdefy\dev\.next\prerender-
14:10:25 ✖ Require stack:
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-dorver\next-server.js
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-dorver\next.js
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-dorver\lib\start-server.js
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-doi\next-start.js
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-dob\commands.js
14:10:25 ✖ - C:\Users\Xqw\Desktop\example\node_modules\.pnpm\next@12.3.1_@babel+core@7.21.4_react-don\next
14:10:25 ✖     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
14:10:25 ✖     at Function.mod._resolveFilename (C:\Users\Xqw\Desktop\example\node_modules\.pnpm\nexact@17.0.2\node_modules\next\dist\build\webpack\require-hook.js:27:32)
14:10:25 ✖     at Function.mod._resolveFilename (C:\Users\Xqw\Desktop\example\node_modules\.pnpm\nexact@17.0.2\node_modules\next\dist\build\webpack\require-hook.js:27:32)
14:10:25 ✖     at Function.Module._load (node:internal/modules/cjs/loader:871:27)
14:10:25 ✖     at Module.require (node:internal/modules/cjs/loader:1098:19)
14:10:25 ✖     at require (node:internal/modules/cjs/helpers:108:18)`

**Comment by Gervwyk** [comment]
Never seen this error before. Please tell us more? What command are you running? Please try running with most basic config, for example:
```yaml
lowdefy: 4.0.0-rc.9
name: test
pages:
 - id: btn
   type: Button
```
